package.json

```diff
@@ -4,19 +4,14 @@
   "description": "",
   "main": "./build/app.js",
   "scripts": {
-    "build": "rimraf build && parallelshell \"npm run build:js\" \"npm run build:style\"",
-    "build:js": "babel src --out-dir build",
-    "build:style": "node-sass --output-style compressed --source-map build/james.css.map style/main.scss build/james.css && postcss --use autoprefixer build/james.css -o build/james.css && npm run copy-files",
-    "watch": "parallelshell \"npm run watch:js\" \"npm run watch:style\"",
-    "watch:js": "nodemon --watch src --ext js --no-stdin --exec \"npm run build:js\"",
-    "watch:style": "nodemon --watch style --ext scss --no-stdin --exec \"npm run build:style\"",
-    "copy-files": "rimraf build/fonts && mkdirp build/fonts && ncp ./node_modules/font-awesome/fonts/ build/fonts/ && ncp ./resource/ ./build/",
-    "package": "npm run build && rimraf ./package &&  mkdirp ./package && mkdirp ./binaries && browserify --node -e build/index.js -o ./package/index.js -u remote -u child-killer && npm run -s copy-files-package && npm run -s create-binaries",
+    "build": "gulp",
+    "watch": "gulp watch",
+    "package": "gulp clean default && mkdirp ./package && mkdirp ./binaries && browserify --node -e build/index.js -o ./package/index.js -u remote -u child-killer && npm run -s copy-files-package && npm run -s create-binaries",
     "create-binaries": "electron-packager ./package James --all --overwrite --icon=./resource/icon.icns --version=0.36.4 --out binaries",
     "zip-binaries": "cd binaries; rm *.zip; for i in *; do zip --symlinks -r \"${i%/}.zip\" \"$i\"; done",
     "copy-files-package": "rimraf package/fonts && mkdirp package/fonts && ncp ./node_modules/font-awesome/fonts/ package/fonts/ && ncp ./resource/ ./package/ && ncp ./build/james.css ./package/james.css",
     "start": "electron .",
-    "lint": "eslint ./src ./test",
+    "lint": "gulp lint",
     "test": "mocha --reporter dot --recursive -r setup-referee-sinon/globals test/unit --compilers js:babel-core/register",
     "test-ci": "npm run lint && npm run test && npm run build && npm run package"
```

Comment:
I want to make packaging and testing occur through `gulp` too (so we don't need `rimraf`, `mkdirp`, etc) but it's a little trickier. No use in delaying faster builds, it can be done separately